tiledb/sm/config/config.cc

```diff
@@ -527,6 +527,25 @@ Status Config::get(const std::string& param, T* value, bool* found) const {
   return Status::Ok();
 }
 
+template <class T>
+T Config::get(
+    const std::string& key,
+    [[maybe_unused]] const Config::MustFindMarker& marker) const {
```

Comment:
You can simply leave out the identifier `marker` and thereby eliminate the need for the attribute.